src/microbe_stage/Microbe.cs

```diff
@@ -282,20 +280,24 @@ public override void _Ready()
                 }
             }
 
+            Mass = Constants.MICROBE_BASE_MASS;
+
```

Comment:
Wouldn't this be fine to be on line 269 still? I don't understand why we shouldn't initialize mass for non save loaded microbes.

Reply:
its strange taht i didnt see any kind of difference with other microbes

Reply:
I think the base mass is so small related to organelle mass that it is very insignificant... that might explain it.